recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/system/xc-installer.te

```diff
@@ -79,9 +79,9 @@ optional_policy(`
 ')
 
 allow xc_installer_t self:fifo_file manage_fifo_file_perms;
-allow xc_installer_t xc_installer_storage_t:dir manage_dir_perms;
-allow xc_installer_t xc_installer_storage_t:file manage_file_perms;
-allow xc_installer_t xc_installer_tmp_t:file manage_file_perms;
-allow xc_installer_t xc_installer_tmp_t:dir manage_dir_perms;
-allow xc_installer_t xc_installer_tmpfs_t:file manage_file_perms;
-allow xc_installer_t xc_installer_tmpfs_t:dir manage_file_perms;
+xc_installer_manage_storage_files(xc_installer_t)
```

Comment:
Don't change this since you've already done it.  But I believe you usually only define interfaces when you "expose" permissions outside the current module.  So `xc_installer_read_tmp_files(lvm_t)` and `xc_installer_read_tmpfs_files(lvm_t)` for lvm, but you can skip the others.